libs/etcd_broker/src/subscription_key.rs

```diff
@@ -0,0 +1,313 @@
+//! Etcd broker keys, used in the project and shared between instances.
+//! The keys are split into two categories:
+//!
+//! * [`SubscriptionFullKey`] full key format: `<cluster_prefix>/<node_kind>/<operation>/<tenant>/<timeline>/<node_id>`
```

Comment:
AFAIR we wanted to place `/<tenant>/<timeline>/` earlier, so we can use a prefix to watch all updates on the selected timeline as a prefix. If `<node_kind>/<operation>` is placed before, than we need to subscribe to multiple prefixes to watch all timeline updates.

Reply:
I will return it back to the beginning, thanks.
Seemed a good solution for both SK and PS subscriptions, but can be improved later.